src/Pos/Delegation/Listeners.hs

```diff
@@ -82,7 +87,7 @@ pskLightRelay = Data $ DataParams $ \pSk -> do
 pskHeavyRelay
     :: WorkMode ssc ctx m
     => Relay m
-pskHeavyRelay = Data $ DataParams $ handlePsk
+pskHeavyRelay = Data $ DataParams MsgTransaction $ \_ _ -> handlePsk
```

Comment:
Well, then it would be good to document these types of messages (maybe they are already documented, but I can't find it, haddock comments [here](https://github.com/serokell/time-warp-nt/blob/master/src/Network/Broadcast/OutboundQueue/Types.hs#L160) don't clarify much). Currently it's not clear which criteria is used to determine the type and which implications this choice has.